js/main.js

```diff
@@ -20,3 +20,25 @@ const lineLenght = function (line, maxLenght) {
 
 lineLenght('', 130 );
 
+const getRandomArrayElement = (elements) => elements[getRandomIntInclusive(0, elements.length - 1)];
+
+const ID = Array.from({length: 25}, (_, index) => index + 1);
+
+const URL = Array.from({length: 25}, (_, index) => `photos/${index + 1}.jpg`);
```

Comment:
Получаем массив, нужно это обозначить. И тут не просто урлы, а урлы картинок.
```suggestion
const ImageURLs = Array.from({length: 25}, (_, index) => `photos/${index + 1}.jpg`);
```